docs/api.rst

```diff
@@ -12122,3 +12119,50 @@ Custom Assets
    :statuscode 200: Custom asset types retrieved successfully.
    :statuscode 409: No user is currently logged in. Check error message.
    :statuscode 500: Internal rotki error.
+
+
+Events Details
+================
+
+.. http:get:: /api/(version)/history/events/details
+
+   Doing a GET on this endpoint will return the details of a history event.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/assets/custom HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {"event_identifier": "0x66e3d2686193e01a4fbf0f598872236402edfe2f4efad84c4f6cdc753b8c78e3", "sequence_index": "222"}
+
+   :reqjson int[optional] event_identifier: The identifier of the event to be queried.
+   :reqjson string[optional] sequence_index: The sequence_index of the event to be queried.
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": {
+              "sub-swaps": [
+                  {"amount_in": "100.0", "amount_out": "0.084", "from_asset": A_SUSHI.identifier, "to_asset": A_ETH.identifier},  # noqa: E501
+                  {"amount_in": "0.084", "amount_out": "98.2", "from_asset": A_ETH.identifier, "to_asset": A_USDT.identifier},  # noqa: E501
+              ],
```

Comment:
the sequence index is optional. How is the response when there are multiples?

Reply:
No, it's not optional, it's required. It was a mistake in the docs